### ISSUE TITLE
Error when injecting fix

### ISSUE THREAD

**Issue by BrianAllred** [Warpten/hzdnptr#1]
Running the given command from an elevated Powershell results in 

```
Error in GetFileVersionInfoSize: 1813
[!] Allocating 500 bytes of data.
[+] Finding kernel32!LoadLibrary.
[+] Remote execution of kernel32!LoadLibraryA failed with error code 0x0.
```

I get the same error in GetFileVersionInfoSize when running from an elevated command prompt, but it still says `Intialization successful.` Sadly, this immediately crashes the game.

**Comment by Warpten** [comment]
Which version are you using? The calls to `GetFileVersionInfoSize` were removed because the game doesn't have a version info block:

![](https://i.imgur.com/tnkYuw4.png)

**Comment by Warpten** [comment]
Fixed by 381f68d12e40b013886f0a446837239e66afd84e

For prebuilt binaries, visit https://ci.appveyor.com/project/Warpten/hzdnptr/builds/34654694/artifacts
